stingray/lightcurve.py

```diff
@@ -591,7 +591,7 @@ def make_lightcurve(toa, dt, tseg=None, tstart=None, gti=None, mjdref=0,
             A :class:`Lightcurve` object with the binned light curve
         """
 
-        toa = np.asarray(toa)
+        toa = np.sort(np.asarray(toa))
```

Comment:
Any particular reason you snuck in a sorting operation here? Does data ever arrive unsorted? Could doing this quietly break anything that the user is doing? Not saying it shouldn't be here, just looking for edge cases. :)

Reply:
Later on there was an operation that made some `time[-1] - time[0]` operation, that broke under some circumstances. Anyway, this method makes a light curve that sorts the event times so I don't think it has any side effects.